eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/EnactSpec.hs

```diff
@@ -61,8 +60,9 @@ relevantDuringBootstrapSpec ::
   ConwayEraImp era =>
   SpecWith (ImpTestState era)
 relevantDuringBootstrapSpec = do
-  actionPrioritySpec
-  hardForkInitiationNoDRepsSpec
+  describe "ENACT (bootstrap phase)" $ do
```

Comment:
It will be better to nest `ENACT` in the `Imp` module
```suggestion
  describe "bootstrap phase" $ do
```

eg.
```haskell
    withImpState @era $ do
      describe "ENACT" $ Enact.relevantDuringBootstrapSpec @era
```

Reply:
Yes! This has been bothering me from the beginning - either not being able to run all enact tests with "ENACT" OR getting  duplicated "ENACT" in the test results - both suck! 
I applied your suggestion and added a "describe RULE" in `Imp` for both versions, so now the results are neat 
![2024-05-08_19-52](https://github.com/IntersectMBO/cardano-ledger/assets/2070003/cbb56d31-50d5-4b26-9844-e414e6c9dee6)